lib/runner/request-helpers-presend.js

```diff
@@ -87,14 +87,14 @@ module.exports = [
 
                 // auth handler gave a no go, and an intermediate request.
                 // make the intermediate request the response is passed to `init` hook
-                replayController.requestReplay(context, item, true, function (err, response) {
+                replayController.requestReplay(context, item, true, function (error, response) {
```

Comment:
Why are there code changes in a documentation only P.R? 😞

Reply:
it's just a variable rename for consistency in the docs
3 of the hooks has the `err` and 1 has `error` parameter

Reply:
since the jsDoc params should match the declaration time params, and callbacks don't have any declarations; this is not needed. Reverting...